lib/full_node.js

```diff
@@ -0,0 +1,14 @@
+//      
+var bcoin = require('bcoin');
+var TrustIsRisk = require('./trust_is_risk');
+
+class FullNode extends bcoin.fullnode {
+                     
+
+  constructor(options         ) {
```

Comment:
The flow-type annotated source code in src/ is compiled to lib/ by [flow-remove-types](https://github.com/flowtype/flow-remove-types), which replaces flow syntax with whitespace to make the code runnable.

Do not review the code in lib.